src/app/home/home.service.ts

```diff
@@ -154,13 +154,7 @@ export class HomeService {
     }
     let unreadMessages = 0;
     let noOfChats = 0;
-    let todoItem: TodoItem = {
-      type: 'chat',
-      name: '',
-      description: '',
-      time: '',
-      meta: {}
-    };
+    let todoItem: TodoItem;
```

Comment:
Why do we change this here?

Reply:
because in dashboard always showing one empty chat notification card if there are no todoItem to show.